datafusion/sqllogictest/test_files/join.slt.part

```diff
@@ -1312,3 +1312,85 @@ SELECT a+b*2,
 
 statement ok
 drop table t1;
+
+
+statement ok
+CREATE TABLE t1(v0 BIGINT, v1 BIGINT);
+
+statement ok
+CREATE TABLE t0(v0 BIGINT, v1 BIGINT);
+
+statement ok
```

Comment:
Thanks!

Could we please add some negative cases to these tests too such as:
1. When there are two subqueries
2. When the subqueries have no correlation

Reply:
only added (2) in https://github.com/apache/datafusion/pull/14595/commits/c353246262f264a06d433b5977994b5d468bad76, I have some test cases for (1) in the comments but I don't plan to support them right now.